src/cljs/main/broadfcui/page/method_repo/method/configs.cljs

```diff
@@ -0,0 +1,231 @@
+            (tab-bar/render-title
+             "CONFIGURATION"
+             (str namespace "/" name))
+            [:div {:style {:paddingLeft "2rem"}}
+             (tab-bar/render-title
+              "SNAPSHOT"
+              (:config-snapshot-id props))]])]
+        (cond
+          config-error
+          [:div {:style {:textAlign "center" :color (:exception-state style/colors)}}
+           "Error loading config: " config-error]
+          (not config)
+          [:div {:style {:textAlign "center" :padding "1rem"}}
+           [comps/Spinner {:text "Loading config..."}]]
+          :else
+          [:div {:style {:display "flex"}}
+           (when owner?
+             [mr-sync/SyncContainer (merge {:ref "sync-container"} config)]
+             (this :-render-sidebar))
+           (this :-render-main)])]))
+   :-render-sidebar
+   (fn [{:keys [state locals refs]}]
+     (let [{:keys [config]} @state
+           {:keys [body-id]} @locals]
+       [:div {:style {:flex "0 0 270px" :paddingRight 30}}
+        (modals/show-modals
+         state
+         {:deleting?
+          [Redactor {:entity config :config? true :on-delete #(nav/go-to-path :method-repo)}]
+          :sharing?
+          [mca/AgoraPermsEditor
+           {:save-endpoint (endpoints/persist-agora-entity-acl true config)
+            :load-endpoint (endpoints/get-agora-entity-acl true config)
+            :entityType (:entityType config)
+            :entityName (mca/get-ordered-name config)
+            :title (str (:entityType config) " " (mca/get-ordered-name config))
+            :on-users-added #((@refs "sync-container") :check-synchronization %)}]})
+        [Sticky
+         {:sticky-props {:data-check-every 1
+                         :data-anchor body-id}
+          :contents
+          [:div {:style {:width 270}}
+           [SidebarButton
+            {:style :light :color :button-primary
+             :text "Permissions..." :icon :settings :margin :bottom
+             :onClick #(swap! state assoc :sharing? true)}]
+           [SidebarButton
+            {:style :light :color :exception-state
+             :text "Redact" :icon :delete :margin :bottom
+             :onClick #(swap! state assoc :deleting? true)}]]}]]))
+   :-render-main
+   (fn [{:keys [state locals this]}]
+     (let [{:keys [config exported-config-id exported-workspace-id blocking-text]} @state
+           {:keys [managers method entityType payloadObject]} config
+           {:keys [body-id]} @locals]
+       [:div {:style {:flex "1 1 auto" :overflow "hidden"} :id body-id}
+        [:div {:style {:display "flex"}}
+         (style/create-summary-block (str "Config Owner" (when (> (count managers) 1) "s"))
+                                     (string/join ", " managers))
+         (style/create-summary-block "Designed For" (str "Method Snapshot " (:snapshotId method)))]
+
+        (style/create-summary-block "Entity Type" entityType)
```

Comment:
Entity Type seems extraneous here. The value will always be "Configuration".

Reply:
That should be showing the workspace entity type that this configuration is designed to run against.